lib/git_reflow/git_helpers.rb

```diff
@@ -58,8 +58,9 @@ def merge_feature_branch(feature_branch_name, options = {})
     end
 
     def append_to_squashed_commit_message(message = '')
-      run "echo \"#{message}\" | cat - .git/SQUASH_MSG > ./tmp_squash_msg"
-      run 'mv ./tmp_squash_msg .git/SQUASH_MSG'
+      git_root_dir = `echo $(git rev-parse --show-toplevel)`.strip
```

Comment:
I don't think you need the `echo`, and we'll need to proxy this to the `run` command.  So this should be something like `git_root_dir = run('git rev-parse --show-toplevel')`